requirements.txt

```
@@ -4,6 +4,6 @@ beautifulsoup4>=4.6.0,<5.0.0
 configparser>=3.5.0,<4.0.0
 keyring>=21.4.0
 requests>=2.13.0,<3.0.0
-fido2>=0.8.1,<=0.9.0
+fido2>=0.9.1,<1.0.0
```

Since some changes in `fido2` from 0.7 to 0.8, and from 0.8 to 0.9 broke `gimme-aws-creds`, to be safe I suggest:

```suggestion
fido2>=0.9.1,<0.10.0
```

---

Sounds good @gcbeltramini - thanks !